Salt master harden

cznewt opened the pull request.

elemoine commented:
I would appreciate a bit more details on why this change was needed, and what problems it fixed. Thanks.

cznewt commented:
It was addressing the issues discussed in integration channel, I'm making multiple changes to model to get it ready to some reasonable state, so we can start next week without any obstacles along the way. For the sysctl i followed https://docs.saltstack.com/en/latest/topics/troubleshooting/master.html . I'll adhere the correct review process when things settle [until sunday at most].

elemoine commented:
> It was addressing the issues discussed in integration channel, I'm making multiple changes to model to get it ready to some reasonable state, so we can start next week without any obstacles along the way. For the sysctl i followed https://docs.saltstack.com/en/latest/topics/troubleshooting/master.html . I'll adhere the correct review process when things settle [until sunday at most].

Thanks. Just trying to understand what you're doing ;)
